src/App/Formatter/HXL/HXLMetadata.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Ushahidi API Formatter for HXL Metadata
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\App\Formatter\HXL;
+
+use Ushahidi\App\Formatter\API;
+use Ushahidi\Core\Traits\FormatterAuthorizerMetadata;
+
+class HXLMetadata extends API
```

Comment:
I've got to figure out how to remove the need for these classes, since they're all the same!